src/Microsoft.OData.Core/Json/IJsonWriterFromStreamFactory.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;

```

Comment:
Do we need a copyright header on this file?

Reply:
Thanks, must have missed this one. We should have something automated for this.